src/vso-task-lib/lib/vsotask.js

```diff
@@ -223,3 +256,8 @@ var _filter = function (pattern, options) {
     return minimatch.filter(pattern, options);
 }
 exports.filter = _filter;
+
+var _findFiles = function (pattern) {
+    return glob.sync(pattern);
```

Comment:
the task lib has a find function.  that function returns an array of files that you then pass into tl.filter.  find uses shelljs find which is actually async under the covers but called in a synchronous fashion.  glob.sync will block the node IO loop.  we also shouldn't add another findFiles method - tl offers find and filter which achieve that.